### PR TITLE
feat: add timeline statistics

### PR DESCRIPTION
**Pull request by maciejBart99**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/valueadd-poland/pimp-my-pr/blob/master/CONTRIBUTING.md#git-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #160 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by maciejBart99** [comment]
image

**Comment by maciejBart99** [comment]
Timeline view - without actual chart
image

**Comment by maciej-lukasik-va** [comment]
image

File: libs/pmp-web/repository/repository-pr-timeline/ui-charts/src/lib/double-axis-chart/double-axis-chart.component.ts [hunk 1]
```diff
@@ -0,0 +1,477 @@
+import {
```
Comment (maciejBart99) [inline libs/pmp-web/repository/repository-pr-timeline/ui-charts/src/lib/double-axis-chart/double-axis-chart.component.ts:1]: This component was based on code from this repo:
https://github.com/habi4ek/double-axis-chart

**Comment by MaciejSikorski** [comment]
image
Here should be the name of the repository.

When changing the repository you see statistics generated for the previous repostiory

**Review by MaciejSikorski — CHANGES_REQUESTED** [review]

**Review by MaciejSikorski — APPROVED** [review]

**Pull request merged**